README.md

```diff
@@ -144,13 +144,20 @@ This will output `squeezenet_v1.1.model` in the current working directory, and i
 
 To learn more about exporting, check out [MMS export documentation](docs/export.md)
 
+## Production Deployments
+
+When launched directly, MMS uses a standalone [Flask](http://flask.pocoo.org/) server.  This is handy for testing and development.
+But for production deployments, we recommend using [Gunicorn](http://gunicorn.org/) which should provide lower latency, higher throughput, and more efficient
```

Comment:
Propose to mention NGINX as well here: "...we recommend using Gunicorn fronted by NGINX which..."